OracleDatabase/dockerfiles/12.1.0.2/installPerl.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Install latest Perl
+cd $INSTALL_DIR
+mv $ORACLE_HOME/perl $ORACLE_HOME/perl.old
+wget http://www.cpan.org/src/5.0/perl-5.14.1.tar.gz
```

Comment:
If you can't guarantee the authenticity of the source tarball via HTTPS, you can't guarantee the authenticity of the checksum from that source either. However, it seems that CPAN don't sign their tarballs either, so there isn't a GPG key to check against.

I'm not sure what the best resolution for this is, to be honest.